permissions/plugin-jobConfigHistory.yml

```diff
@@ -6,4 +6,5 @@ paths:
 - "org/jvnet/hudson/plugins/jobConfigHistory"
 developers:
 - "jochenafuerbacher"
-- "stefanbrausch"
\ No newline at end of file
+- "stefanbrausch"
+- "robinschulz"
```

Comment:
Is your ID correct, Not robin**r**schulz?

Reply:
The ID in the file should be the Jenkins infra user id (e.g., the username used to login to Jira and friends).